app/src/main/java/com/example/smarteden/greenhousefields/RecyclerViewAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package com.example.smarteden.greenhousefields
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.example.smarteden.R
+class CardViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+
+    val titleTextView= itemView.findViewById<TextView>(R.id.textview_title)
+    val subtitleTextView = itemView.findViewById<TextView>(R.id.textview_subtitle)
+    val descriptionTextView = itemView.findViewById<TextView>(R.id.textview_description)
+
```

Comment:
Für wos stehen do eigentlich title, subtitle und description? 🙈 
Und wos für Daten lodsd du do eine? Werd ma wahrscheinlich a Viewmodel mit Live Data braucha

Reply:
Ja des is ja etz blos aktuell drinnen des werdi no alles verändern nur dasi de basics scho mal drin hab